src/views/authenticated/List.tsx

```diff
@@ -1,3 +1,4 @@
+import { useParams } from "react-router-dom";
```

Comment:
On line 6 `useNavigate` is imported from `react-router-dom` again and I get an error in the terminal from vite about dupilicate imports. We should be able to combine the 2 imports to stop that error.

```suggestion
import { useParams, userNavigate } from "react-router-dom";
```